JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1427,4 +1447,68 @@ Descriptives <- function(jaspResults, dataset, options) {
     return(TRUE)
   else
     return(FALSE)
-}
\ No newline at end of file
+}
+
+.descriptivesStemAndLeaf <- function(container, dataset, variables, options) {
```

Comment:
Naming can be improved though. Does this make/compute/plot a stem and leaf object? Perhaps add a verb to the function to emphasise the action.